src/params/str-list.js

```diff
@@ -10,6 +10,18 @@ class StrList extends List {
       assert(opts.enums.every(e => typeof e === 'string'));
       this.enums = new Set(opts.enums);
     }
+    if (opts.minListItemLength !== undefined) {
```

Comment:
Doesn't need to be an `if` check.
You're doing the `this.minLIstItemLength !== undefined` in the validation, so it's okay to assign it
```suggestion
  assert(opts.minListItemLength === undefined || Number.isInteger(opts.minListItemLength));
  this.minListItemLength = opts.minListItemLength;
```

The message you're conveying here with the `assert` is that `minListItemLength` can be `undefined` or an `integer`